Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Customers.h

```diff
@@ -36,7 +37,7 @@
  *  @param customer A BUYCustomer
  *  @param error    An optional NSError
  */
-typedef void (^BUYDataCustomerBlock)(BUYCustomer *customer, NSError *error);
+typedef void (^BUYDataCustomerBlock)(BUYCustomer * _Nullable customer, NSError * _Nullable error);
```

Comment:
We could make these declarations easier to parse by using assume nullable macros around them, and move the assume nonnull begin macro further down. Just a suggestion, though. I'm 50/50 on which is better.